osu.Game/Overlays/NotificationOverlayToastTray.cs

```diff
@@ -24,16 +24,16 @@ namespace osu.Game.Overlays
     /// </summary>
     public partial class NotificationOverlayToastTray : CompositeDrawable
     {
-        public override bool IsPresent => toastContentBackground.Height > 0 || toastFlow.Count > 0;
+        public override bool IsPresent => toastContentBackground.Height > 0 || Notifications.Any();
```

Comment:
I don't get this change. I can get that the public `Notifications` member was not necessarily correct from a data point of view, therefore causing `NotificationOverlay.AllNotifications` to _also_ be wrong, but why are changes being done here? The old code looks more correct to me. For stuff like presence I think the visual state is more correct rather than the "actual" state wherein some notifications are in transition?

Reply:
The notification is still being displayed by the toast tray, albeit in a different place than `toastFlow`, so it looks more correct for `IsPresent` to be true when there are notifications that are undergoing the forward transition. This is not making a difference because there's the `toastContentBackground.Height > 0` conditional next to it.

All that being said, I'm just going to revert these changes because it deviates the point of this PR entirely and I don't think that's desired.